content/team-current/neelima.md

```diff
@@ -0,0 +1,11 @@
++++
+
+date = "2021-06-16T19:21:02+04:30" 
+name = "Neelima Boppana"
+image = "neelima.png"
+linkedin = "www.linkedin.com/in/NeelimaBN"
```

Comment:
@NeelimaBN Please update this to include the fqdn.
It must be `https://www.linkedin.com/in/NeelimaBN`